example/passthrough/README.md

```diff
@@ -0,0 +1,45 @@
+# "Passthrough" setup for OpenTelemetry
+
+Some go programs may wish to propagate context without recording spans.  To do this in OpenTelemetry, simply install `TextMapPropagators`, but do not install a TracerProvider using the SDK.  This works because the default TracerProvider implementation returns a "Non-Recording" span that keeps the context of the caller, but does not record spans.
```

Comment:
```suggestion
Some Go programs may wish to propagate context without recording spans. To do this in OpenTelemetry, simply install `TextMapPropagators`, but do not install a TracerProvider using the SDK. This works because the default TracerProvider implementation returns a "Non-Recording" span that keeps the context of the caller but does not record spans.
```